src/types.ts

```diff
@@ -32,7 +33,7 @@ export interface ICourierSendParameters {
   recipientId: string;
   data?: object;
   brand?: string;
-  preferences?: ICourierProfilePreferences;
+  preferences?: ICourierClientPreferences;
```

Comment:
@scarney81 This is not right. Should it just go back to object? Has it been updated to accept this format?

Reply:
I moved it back to use `ICourierProfilePreferences` because that's the old way. If the send has been updated then we should update it.

Reply:
@aydrian - The type should actually be:

```
export interface IProfilePreferences {
  categories: {
    [categoryId: string]: {
      status: PreferenceStatus;
    };
  };
  notifications: {
    [notificationId: string]: {
      status: PreferenceStatus;
    };
  };
}
```

Reply:
So I should update ICourierPreferencesPutParameters and use that in the other places as well

Reply:
@aydrian - this is not for `put` this is for the `send` endpoint